### ISSUE TITLE
Style/RedundantInterpolation: auto-correction adds unnecessary parentheses for single method call

### ISSUE THREAD

**Issue by deepj** [rubocop-hq/rubocop#8835]
## Expected behavior

Auto-corrected `"#{number}"` => `number.to_s`

## Actual behavior

Auto-corrected `"#{number}"` => `(number).to_s`

Auto-corrected code violates `Style/RedundantParentheses` cop.

## Steps to reproduce the problem

```ruby
      if name.present?
        "#{number} [#{name}]"
      else
        "#{number}"
      end
```

Run
```
rubocop --only Style/RedundantInterpolation -a
```

## RuboCop version

```
$ rubocop -V
0.92.0 (using Parser 2.7.1.5, rubocop-ast 0.7.1, running on ruby 2.7.1 x86_64-darwin18)
```
